src/App.js

```diff
@@ -1,29 +1,42 @@
-import logo from './logo.svg';
+import {
+  createBrowserRouter,
+  RouterProvider,
+  Route,
+  createRoutesFromElements,
+} from 'react-router-dom';
+// import { ToastContainer } from 'react-toastify';
+import Reserve from './components/Reserve';
+import Doctors from './components/Doctors';
+import Doctor from './components/Doctor';
 import './App.css';
+import SignUp from './components/auth_pages/Signup';
+import Login from './components/auth_pages/Login';
+import MyReservations from './components/MyReservations';
+import AddDoctor from './components/AddDoctor';
+import DeleteDoctor from './components/DeleteDoctor';
+import Root from './components/Root';
+import ReservationForm from './components/ReservationForm';
+
+const router = createBrowserRouter(
```

Comment:
- [ ] When trying to create a doctor, I cannot determine the correct way to add the dates:

![image](https://github.com/mailsg/medimate-frontend/assets/57421823/fca0eb8c-5c4e-47ef-953f-013eeb08074d)

Kindly, include some time of date selector or include the correct format.

That way, your users will be able to fill the form easily, and the UX will be vastly improved. 👍